cpp/tests/c_api/similarity_test.c

```diff
@@ -161,15 +161,16 @@ int test_jaccard()
 int test_weighted_jaccard()
```

Comment:
Need to also update `mg_similarity_test.c`

Reply:
Updated.